src/util/string.h

```diff
@@ -492,6 +484,39 @@ inline std::basic_string<T> unescape_string(std::basic_string<T> &s)
 	return res;
 }
 
+/**
+ * Remove all escape sequences in \p s.
+ *
+ * @param s The string in which to remove escape sequences.
+ * @return \p s, with escape sequences removed.
+ */
+template <typename T>
+std::basic_string<T> remove_enriched_text_escapes(const std::basic_string<T> &s) {
```

Comment:
Yeah this is way too long of a name... I can't really offer a suggestion for an alternative, but this is too long.  Also, the line is way over 80 characters.  The { should be on the next line too.